### PR TITLE
Odin_AdminDashboard

### PR DESCRIPTION
**Pull request by aatiqDev1443**
**Project: Odin Admin Dashboard**
 
Live => https://aatiqjalanswe.github.io/Odin-Admin-Dashboard/

**Development:**

1. **Structure**

> HTML File consisting of code structure and relevant links to the external files used i.e images , svg.

2. **Styling**

> CSS File consisting of relevant styling to make the dashboard pleasant and accessible to the user.

**UI:**

image

**Meta Info**

Reviewee: **@aatiqJalanSWE** 
Reviewer: **@devanuj97** **@balwant-jtc**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+            <p><strong>Communities</strong></p>
+          </div>
+          <div class="list-items-bottom">
+            <div class="icon-text">
+              <svg
+                style="width: 20px; height: 20px"
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 24 24"
+              >
+                <title>cog-outline</title>
```
Comment (balwant-jtc) [inline index.html]: `title` is for humans, so just use plain english here without any hyphen "-"

**Review by balwant-jtc — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (aatiqDev1443) [reply index.html]: fixed it, sir.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+  <body>
+    <div class="grid">
+      <div class="sidebar">
+        <div class="logo-text">
+          <svg
+            style="width: 40px; height: 40px"
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 24 24"
+          >
+            <title>ViewDashboard</title>
```
Comment (balwant-jtc) [inline index.html]: title should be in plain english like "view dashboard"

**Review by balwant-jtc — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (aatiqDev1443) [reply index.html]: fixed it sir
